src/components/Pages/home/DoctorDetails.js

```diff
@@ -0,0 +1,165 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import {useNavigate} from 'react-router-dom';
+import React from "react";
+
+const DoctorDetailsCtn = styled.div`
+  @media (min-width: 768px) {
+    height: 100%;
+    width: 400px;
+    position: fixed;
+    top: 0;
+    right: 0;
+    background-color: var(--white-sobber);
+    z-index: 1;
+    transform: translateX(${(props) => (props.detailsOpen ? 0 : 400)}px);
+    visibility: ${(props) => (props.detailsOpen ? 'visible' : 'hidden')};
+    transition: 200ms ease-in-out 0s;
+
+    .doc-info-ctn {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+
+    .img-ctn {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      margin-top: 50px;
+      align-self: center;
+      width: 150px;
+      height: 150px;
+    }
+
+    .img-bg {
+      width: 150px;
+      height: 150px;
+      margin: auto;
+      border-radius: 100%;
+      background: var(--img-background);
+    }
+
+    .doc-img {
+      width: 150px;
+      height: 150px;
+      margin: auto;
+      border-radius: 100%;
+    }
+
+    .doc-name {
+      align-self: center;
+    }
+
+    .doc-info-tbl {
+      width: 100%;
+      display: flex;
+      justify-content: center;
+      flex-direction: column;
+    }
+
+    .doc-info-row {
+      display: flex;
+      justify-content: space-between;
+      background: var(--white-sobber);
+      padding: 0 30px;
+      font-family: "Ubuntu Light";
+      color: var(--text-black);
+      font-size: 1rem;
+    }
+
+    .white {
+      background: rgb(249, 255, 247);
+    }
+
+    .reserve-btn {
+      background-color: var(--green);
+      color: white;
+      align-self: center;
+      padding: 14px 20px;
+      width: 200px;
+      border: none;
+      border-radius: 4px;
+      cursor: pointer;
+      transition: all 300ms linear 0s;
+      margin-top: 50px;
+
+      :hover {
+        background-color: var(--green-darker);
+      }
+    }
+  }
+`;
```

Comment:
SEe previous comment on same

Reply:
Hi @mcrd25, thank you for your review. This is actually a styled component to make things reusable and been able to pass props to make the design dynamic. We found it really useful so we decided to use it inside our project

Reply:
But we could remove it.

Reply:
Hey, this is not a problem, however, would it not be easier for this to be a class in CSS file? and change class? also would make for cleaner code - you are free to keep it this way but it's not the cleanest way to do this